recipes/jsonpointer/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "jsonpointer" %}
+{% set version = "1.10" %}
+{% set sha256 = "9fa5dcac35eefd53e25d6cd4c310d963c9f0b897641772cd6e5e7b89df7ee0b1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - six
+
+  run:
+    - python
+    - six
+
+test:
+  imports:
+    - jsonpointer
+
+about:
+  home: https://github.com/stefankoegl/python-json-pointer
+  license: BSD
```

Comment:
BSD-3-Clause here.
Can you make the same changes that @synapticarbors mentioned in `jsonpatch` PR?